f3dex2.s

```diff
@@ -448,13 +517,13 @@ OSTask:
 .create CODE_FILE, 0x00001080
 
 // Global registers
-curClipRatio equ $13
 cmd_w1 equ $24
 cmd_w0 equ $25
 taskDataPtr equ $26
 inputBufferPos equ $27
 rdpCmdBufPtr equ $23
 rdpCmdBufEnd equ $22
+vOne equ $v1
```

Comment:
If there's a `vOne` name for $v1, then I think there should also be a `vZero` name for $v0 for parity.

Reply:
I had not made that change because it would be a lot of lines changed, and also because I thought users would be more likely to know that register 0 holds zeroes from MIPS (even though `$v0` is a real register which doesn't have to hold zero). But since you prefer, I've done the replacement.